.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-        ],
-        "actions": [
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "First Time Contributor :wave:"
-            }
-          },
-          {
-            "name": "addReactionToIssue",
-            "parameters": {
-              "reaction": "heart"
-            }
-          },
-          {
-            "name": "addReactionToIssue",
-            "parameters": {
-              "reaction": "hooray"
-            }
-          },
-          {
-            "name": "addReply",
-            "parameters": {
-              "comment": "${issueAuthor} Welcome to the Windows Community Toolkit 💙💙💙\n\nWe always love to see new contributors joining the community and taking initiatives to contribute and add value to the toolkit 🙌\n\nIf you haven't had a chance to explore our [Contribution Wiki page](https://github.com/windows-toolkit/WindowsCommunityToolkit/wiki) then please feel free to do as it will help you transition smoothly.\n\nOnce again Thank you so much for the contribution and we are certainly looking forward to working with you more in the future 🚀🚀"
-            }
-          }
-        ],
-        "taskName": "Welcome message to the first time Github Committer when opening a new PR"
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "E8apbGccM",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isOpen",
-              "parameters": {}
-            },
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Bugs 7.1",
-                    "columnName": ""
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Features 7.1"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Technical 7.1"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Samples 7.1"
-                  }
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "actions": [
-          {
-            "name": "addMilestone",
-            "parameters": {
-              "milestoneName": "7.1"
-            }
-          }
-        ],
-        "taskName": "Add milestone if the issue is added in the project column (7.1)"
```

Comment:
Project board automation is not supported in the new service, so related tasks were not ported.